src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/DAC/DacFields/PX1014/NonNullableTypeForBqlFieldTests.cs

```diff
@@ -17,46 +17,30 @@ namespace Acuminator.Tests
 {
     public class NonNullableTypeForBqlFieldTests : CodeFixVerifier
     {
-	    private DiagnosticResult CreatePX1014DiagnosticResult(int line, int column)
-	    {
-			var diagnostic = new DiagnosticResult
-			{
-				Id = Descriptors.PX1014_NonNullableTypeForBqlField.Id,
-				Message = Descriptors.PX1014_NonNullableTypeForBqlField.Title.ToString(),
-				Severity = DiagnosticSeverity.Error,
-				Locations =
-					new[] {
-						new DiagnosticResultLocation("Test0.cs", line, column)
-					}
-			};
-
-		    return diagnostic;
-	    }
-
 	    [Theory]
-	    [EmbeddedFileData(@"Dac\PX1014\Diagnostics\NonNullableTypeForBqlField.cs")] 
+	    [EmbeddedFileData(new[] { @"Diagnostics\NonNullableTypeForBqlField.cs" }, true)] 
```

Comment:
This is not convenient; consider creating several constructors with a different number of arguments.
But again, folder structure should be discussed first.

Reply:
@dropsonic I agree that we need to discuss it in details, this is a prototype, based on existing  proposals - to group together analyzer and fix, sources and code.